content/sensu-go/5.20/api/overview.md

```diff
@@ -612,6 +612,14 @@ curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN http://127.0.0.1:8080/api/cor
 --data-urlencode 'fieldSelector=silenced.namespace != "default"'
 {{< /highlight >}}
 
+To list all silenced events for all namespaces:
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN http://127.0.0.1:8080/api/core/v2/events -G \
+--data-urlencode 'fieldSelector=event.is_silenced == true'
```

Comment:
Looks like this command is not working locally, I think it needs to look like this:

```
curl -H "Authorization: Key $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/events -G \
--data-urlencode 'fieldSelector=event.is_silenced == true'
```

Reply:
In the cURL examples in our docs we are using `Bearer` with the assumption of auth tokens. If you use API key then you have to change to `Key` as you said, but because we are assuming tokens it's still `Bearer`.

Or do you mean it's not working with `Bearer` and an auth token?

Reply:
Oh I see, I missed that. I feel like we should probably push for API tokens instead of auth tokens, but that's another subject.

That being said, there's still a `"` missing after `$SENSU_ACCESS_TOKEN`; it should be:

```
curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/events -G \
--data-urlencode 'fieldSelector=event.is_silenced == true'
```

Reply:
Thank you! I will see about switching to API keys and in the meantime make this change.